src/main/webapp/script.js

```diff
@@ -38,11 +38,18 @@ function display(data, state, county) {
         results.innerHTML = "Location not found.";
     }
```

Comment:
And last but not least, the javascript changes. Just a few to work with the new data we get from the java program, for example, the new fields "avgCases" and "avgDeaths."